app/src/main/java/com/kickstarter/services/DiscoveryParams.java

```diff
@@ -451,6 +452,15 @@ public boolean shouldIncludeFeatured() {
     }
   }
 
+  /**
+   * Determines if params are for All Projects ("Everything"), i.e. discovery without params.
```

Comment:
I don't have the latest iOS app compiling, but is the filter now 'All Projects' instead of 'Everything'?

If so, we could update the string on Android and remove references to 'Everything' throughout the source.